tools/modbam2bed/compile.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Cribbed from htslib example
```

Comment:
This was intended, James developed the functionality in his fork. At some point in the future it will change to using samtools/htslib, probably in the next release in modbam2bed.